literal/src/escape.rs

```diff
@@ -385,7 +385,7 @@ impl<'a> Escape for AsciiEscape<'a> {
     fn layout(&self) -> &EscapeLayout {
         &self.layout
     }
-
+    #[allow(unsafe_code)]
```

Comment:
This is the last unsafe. You can now reference RustPython parser without having to change our clippy configuration

Reply:
That's great!